sql-ballerina/types.bal

```diff
@@ -1115,9 +1115,9 @@ public class ProcedureCallResult {
         return getNextQueryResult(self);
     }
 
-    # Closes the `ProcedureCallResult` object and releases resources.
+    # Closes the `sql:ProcedureCallResult` object and releases resources.
     #
-    # + return - `Error` if any error occurred while closing.
+    # + return - `Error` if any error occurred while closing
```

Comment:
Should be `sql:Error`, right?

Reply:
added in https://github.com/ballerina-platform/module-ballerina-sql/pull/228/commits/d3df4347c964f2a634909920ca548fefdfc1139c